igwn_auth_utils/requests.py

```diff
@@ -0,0 +1,289 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Cardiff University
+# Distributed under the terms of the BSD-3-Clause license
+
+"""Python Requests interface with IGWN authentication
+
+This is heavily inspired by Leo Singer's excellent `requests-gracedb` package.
```

Comment:
Thank you. This acknowledgement means a lot to me! Please add a link.

Reply:
Is this the best reference: https://requests-gracedb.readthedocs.io/?

Reply:
This URL is included in the `intersphinx` configuration, so the references in the docstrings are automatically translated to hyperlinks in the docs.

Reply:
OK, would you please add an intersphinx ref in the docstring here?